code/_onclick/hud/map_popups.dm

```diff
@@ -62,6 +62,11 @@
 	layer = GAME_PLANE
 	plane = GAME_PLANE
 
+/atom/movable/screen/Destroy()
+	master = null
+	//QDEL_NULL(hud) // Not currently ever used; not sure if it would just need to be nulled or qdel
```

Comment:
```suggestion
	hud = null
```
The HUD owns this object, not the other way around, so we null instead of qdeling